Hive.NET.sln

```diff
@@ -1,12 +1,16 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# 
+#
 Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Hive.NET.Core", "Hive.NET.Core\Hive.NET.Core.csproj", "{C46DC3D0-ECA6-43C9-91EF-CEF484B515B7}"
 EndProject
 Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Hive.NET.Demo.Console", "Hive.NET.Demo.Console\Hive.NET.Demo.Console.csproj", "{E2F91C61-B769-4601-BD98-0CDF734403F1}"
 EndProject
 Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Hive.NET.Tests", "Hive.NET.Tests\Hive.NET.Tests.csproj", "{7A6FA868-3C6C-4D32-8833-26099BB115E1}"
 EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "WebApplication1", "WebApplication1\WebApplication1.csproj", "{0EFFF39D-BB53-4988-867F-C1C33D1DAD1C}"
```

Comment:
Some leftovers? Probably can be removed. Project is present in sln on open but files don't exist

Reply:
done